sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -195,20 +198,36 @@ private static PodSpec from(
         if (!(placementRule instanceof PassthroughRule)) {
             builder.placementRule(placementRule);
         }
-        if (rawPod.getContainer() != null) {
+        
+        RawContainerInfoProvider cip = null;
```

Comment:
There's no style guide, other than that provided by suggested variable names by IntelliJ.  Maybe `containerInfoProvider` instead of `cip`.  Prefer more obviously readable variable names.